backend/src/services/mempool_monitor.py

```
@@ -1,224 +1,268 @@
-                                    'details': {'account': str(account_to_close)}
-                                })
-
-                    except Exception as parse_e:
-                        logger.warning(f"Error parsing token instruction for rugpull check: {parse_e}")
-
-        except RPCException as tx_e:
-            logger.error(f"RPC error fetching transaction {signature} for rugpull check: {tx_e}")
-        except Exception as tx_e:
-            logger.error(f"Error processing transaction {signature} for rugpull check: {tx_e}")
+                        if asyncio.iscoroutinefunction(callback):
+                            asyncio.create_task(callback(alert_data))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)